integration/spark-common/src/main/scala/org/apache/spark/sql/execution/streaming/CarbonAppendableStreamSink.scala

```diff
@@ -122,7 +122,7 @@ class CarbonAppendableStreamSink(
         className = sparkSession.sessionState.conf.streamingFileCommitProtocolClass,
         jobId = batchId.toString,
         outputPath = fileLogPath,
-        isAppend = false)
+        false)
```

Comment:
Keep back the old code

Reply:
in 2.3 isAppend name is changed to dynamicPartitionOverwrite